src/node/handler/PadMessageHandler.js

```diff
@@ -1009,6 +1009,8 @@ function handleClientReady(client, message)
   var currentTime;
   var padIds;
 
+  hooks.callAll("clientReady", message.padId);
```

Comment:
Hi Pita. Good point--I've made the changes. Let me know if there's anything else you'd like me to do.